src/scss/03_molecules/_accordionGroup.scss

```diff
@@ -1,15 +1,10 @@
-.m-accordion-group {
-  color: color(text);
-  display: flex;
-  flex: 1 0 auto;
-  flex-direction: column;
-  justify-content: flex-start;
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-  
-  &.m-accordion-group--dark {
-    background: color(backgroundAlt, base);
-    color: color(textAlt, light);
+.m-accordionGroup {
+  .a-accordion:not(:last-child) {
```

Comment:
`.m-accordionGroup .a-accordion:not(:last-child) .a-accordion__toggle {`

`.m-accordionGroup .a-accordion.a-accordion--dark:not(:last-child) .a-accordion__toggle {`